backend/controllers/metricsController.js

```diff
@@ -59,7 +59,7 @@ metricsController.getMemory = async () => {
       .reduce((pods, { values, metric: { pod: podId } }) => {
         let memory = values[0][1];
 
-        if (values[0][1] > 0 && podId) {
+        if (podId) {
```

Comment:
removed the condition because its probably better to be consistent and keep all the pods on the graph.